mautic/mautic
# Keep the filename when uploading

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [&cross;]
| New feature/enhancement? (use the a.x branch)      | [&check;]
| Deprecations?                          | [&cross;]
| BC breaks? (use the c.x branch)        | [&cross;]
| Automated tests included?              | [&cross;] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The filename of the uploaded file via the froala editor is a randomly generated string. This PR fixes it.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Disable GrapesJS plugin.
3. Try to upload an image in the legacy builder.
4. Check if the name is kept.
